auto-generated-sdk/docs/FIJobSettings.md

```diff
@@ -12,6 +12,7 @@ Name | Type | Description | Notes
 **BankLoans** | [**FIBankLoans**](FIBankLoans.md) |  | [optional] 
 **MunicipalBonds** | [**FIMunicipalBondsForJobSettings**](FIMunicipalBondsForJobSettings.md) |  | [optional] 
 **MarketEnvironment** | [**FIMarketEnvironment**](FIMarketEnvironment.md) |  | [optional] 
+**StructuredProducts** | [**FIStructuredProductsForJobSettings**](FIStructuredProductsForJobSettings.md) |  | [optional] 
```

Comment:
We need to cover FI changes in test cases as well.

Reply:
This is quant related changes and it is on priority. So can't take FI changes now will see it later.

Reply:
ok please make a note and handle this on separate PR